interface/globals.php

```diff
@@ -289,45 +292,45 @@ function undoMagicQuotes($array, $topLevel=true) {
       $GLOBALS[$gl_name] = $gl_value;
     }
   }
-  
-  # Put this here to default the globals entry for concurrent_layout while that code is being removed 
+
+  # Put this here to default the globals entry for concurrent_layout while that code is being removed
   $GLOBALS['concurrent_layout'] = 3;
 
   // Language cleanup stuff.
   $GLOBALS['language_menu_login'] = false;
   if ((count($GLOBALS['language_menu_show']) >= 1) || $GLOBALS['language_menu_showall']) {
     $GLOBALS['language_menu_login'] = true;
   }
-  
-  
+
+
 // Additional logic to override theme name.
 // For RTL languages we substitute the theme name with the name of RTL-adapted CSS file.
     $rtl_override = false;
     if( isset( $_SESSION['language_direction'] )) {
-        if( $_SESSION['language_direction'] == 'rtl' && 
+        if( $_SESSION['language_direction'] == 'rtl' &&
         !strpos($GLOBALS['css_header'], 'rtl')  ) {
 
             // the $css_header_value is set above
             $rtl_override = true;
         }
-    }     
-    
-    else { 
+    }
+
+    else {
         //$_SESSION['language_direction'] is not set, so will use the default language
         $default_lang_id = sqlQuery('SELECT lang_id FROM lang_languages WHERE lang_description = ?',array($GLOBALS['language_default']));
-        
+
         if ( getLanguageDir( $default_lang_id['lang_id'] ) === 'rtl' && !strpos($GLOBALS['css_header'], 'rtl')) { // @todo eliminate 1 SQL query
             $rtl_override = true;
         }
     }
-    
+
 
     // change theme name, if the override file exists.
     if( $rtl_override ) {
         // the $css_header_value is set above
         $new_theme = 'rtl_' . $temp_css_theme_name;
 
-        // Check file existance 
```

Comment:
existence, not "existance"  If you correct whitespace, might as well correct everything...

Reply:
Atom corrects whitespace automatically...